cli/src/main/kotlin/org/partiql/pipeline/AbstractPipeline.kt

```diff
@@ -99,6 +100,7 @@ internal sealed class AbstractPipeline(open val options: PipelineOptions) {
     /**
      * Wraps the PartiQLCompilerPipeline
      */
+    @OptIn(PartiQLExperimental::class)
```

Comment:
I wouldn't block on this PR, but it might be worth adding an example of how to consume the experimental pipeline — this being an example but it's buried in the cli package

Reply:
Will create an issue for this to track.

Reply:
https://github.com/partiql/partiql-lang-kotlin/issues/915#issue-1471779872